src/modules/commander/commander_params.c

```diff
@@ -202,6 +202,17 @@ PARAM_DEFINE_FLOAT(COM_RC_LOSS_T, 0.5f);
  */
 PARAM_DEFINE_FLOAT(COM_RCL_ACT_T, 15.0f);
 
+/**
+ * Home position enabled
+ *
+ * Set home position automatically if possible.
+ *
+ * @group Commander
+ * @reboot_required true
+ * @boolean
+ */
+PARAM_DEFINE_INT32(COM_HOME_EN, 1);
```

Comment:
Should we combine this to prevent flying without HOME as well, e.g.:

- 0: home disabled (can't be set)
- 1: home can be set and is set automatically on arming
- 2: home has to be valid in order to arm

Or should these be separate params?

Reply:
For the arming requirements I was thinking about collecting them all as bits in a single parameter, something like COM_ARM_REQ.

Combining `COM_HOME_IN_AIR` might also make sense.